projects/ion/src/lib/icon/svgs/iconsText/doc-check.ts

```diff
@@ -0,0 +1,7 @@
+export default `<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Please remove the svg tags in the files, it is already added in the icon component

Reply:
Removed the svg tags from all 3 files @jairochaves2